src/babylon.engine.ts

```diff
@@ -1305,6 +1305,64 @@
             this.wipeCaches();
         }
 
+        // UBOs
+        public createUniformBuffer(elements: number[] | Float32Array): WebGLBuffer {
+            var ubo = this._gl.createBuffer();
+            this.bindUniformBuffer(ubo);
+
+            if (elements instanceof Float32Array) {
+                this._gl.bufferData((<WebGL2RenderingContext>this._gl).UNIFORM_BUFFER, <Float32Array>elements, this._gl.STATIC_DRAW);

```

Comment:
To avoid this I recommend updating the basic WebGLRenderingContext like I did for other webgl2 features (see mixins.ts files)